src/subscription_manager/managercli.py

```diff
@@ -1140,9 +1141,8 @@ def _do_command(self):
             except ssl.SSLError as e:
                 # since the user can override serverurl for register, a common use case is to try to switch servers
                 # using register --force... However, this normally cannot successfully unregister since the servers
-                # are different... So, do not map the message as SSLError is more useful than its mapped message in
-                # debugging this use case (mapped message is misleading in this use case)
-                handle_exception("Unregister failed: %s", e, map_message=False)
+                # are different.
+                handle_exception("Unregister failed: %s", e, map_message=True)
```

Comment:
With this change, there are no instances of handle_exception called with `map_message=False`, so please remove the `map_message` logic introduced in 217c3863.